ui/src/main/resources/Meeting/Code/NextMeetingsMacro.xml

```diff
@@ -431,6 +444,9 @@
     <property>
       <contentType>No content</contentType>
     </property>
+    <property>
+      <defaultCategories/>
+    </property>
     <property>
       <defaultCategory/>
     </property>
```

Comment:
`defaultCategory` property can be removed as it is replaced by `defaultCategories`.